src/world/Server/WorldSession.cpp

```diff
@@ -283,7 +283,8 @@ void WorldSession::LogoutPlayer(bool Save)
                     case TYPEID_UNIT:
                     {
                         if (const auto creature = dynamic_cast<Creature*>(obj))
-                            creature->loot.looters.erase(_player->getGuidLow());
+                            ;
+                            //creature->loot.looters.erase(_player->getGuidLow());
```

Comment:
?

Reply:
happend when i commented stuff out and forogt to re add ^^